README.md

```diff
@@ -418,6 +418,46 @@ data, mask = multi_part(assets, bbox, ...)
 data, mask = multi_preview(assets, ...)
 ```
 
+#### Reading asset with a GeoJSON Polygon
+
+Natively rio-tiler support mostly `bbox` reading. Using GDALWarpVRT **Cutline** option, it's possible to read a dataset for a given polygon.
+
+```python
+
+from rio_tiler.io import COGReader
+from rio_tiler.utils import create_cutline
+from rasterio.features import bounds as featureBounds
+
+feat =     {
+    "type": "Feature",
+    "properties": {},
+    "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+        [
+        [-52.6025390625, 73.86761239709705],
+        [-52.6025390625, 73.59679245247814],
+        [-51.591796875, 73.60299628304274],
+        [-51.591796875, 73.90420357134279],
+        [-52.4267578125, 74.0437225981325],
+        [-52.6025390625, 73.86761239709705]
+        ]
+    ]
+    }
+}
+
+# Get BBOX of the polygon
+bbox = featureBounds(feat)
+
+# Use COGReader to open and read the dataset
+with COGReader(COGEO) as cog:
+    # Create WTT Cutline
+    cutline = create_cutline(cog.dataset, feat, geometry_crs="epsg:4326")
+
+    # Read part of the data (bbox) and use the cutline to mask the data
+    data, mask = cog.part(bbox, warp_vrt_option={'cutline': cutline})
```

Comment:
Is there a reason to use `cog.part` instead of `cog.tile`? GDAL isn't efficient for a `cutline`?

Reply:
I don't get that, you can use cutline for tile, preview or part (also for metadata), because at the end the warp_vrt_options is passed to the _read function